bokehjs/src/lib/core/ui_events.ts

```diff
@@ -366,6 +366,7 @@ export class UIEventBus {
           this._current_pan_view.on_pan(event)
         } else if (event_type == "pan:end") {
           this._current_pan_view.on_pan_end(event)
+          this.set_cursor(this._current_pan_view.cursor(event.sx, event.sy))
```

Comment:
If you remove this line, here's what happens. When you click inside a box selection and start dragging, the cursor changes from "grab" to "grabbing" but when you stop dragging, instead of the cursor going back to "grab" it stays in the "grabbing" state.